content/docs/specs/native_histograms.md

```diff
@@ -1861,19 +1858,26 @@ native histogram:
 - `idelta()` (For gauge histograms.)
 - `irate()` (For counter histograms.)
 
-TODO: `idelta` and `irate` are not yet implemented for histograms.
-
 These functions SHOULD be applied to either gauge histograms or counter
 histograms as noted above. However, they all work with both flavors, but if at
 least one histogram of an unsuitable flavor is contained in the range vector, a
 warn-level annotation is added to the result.
 
 All these functions return no result for series that contain a mix of float
 samples and histogram samples within the range. A warn-level annotation is
-added for each output element missing for that reason.
+added for each output element missing for that reason. `idelta()` and `irate()`
+only consider the last two samples in the range for this check.
```

Comment:
Athough the last line "`idelta()` and `irate()` only consider the last two samples in the range for this check." clarifies that we are only considering the last two samples, still I think it would be better to have this conveyed in the initial sentence.

Reply:
The problem here is that the initial sentence is the general one that is valid for all these function. The sentence about idelta and irate is the exception to it for these two special functions. Maybe I can munge the last and first sentence together…

Reply:
How do you like it now?

Reply:
Now it looks good to me!
Thanks for the change.